### PR TITLE
Adds New Aquarium Fish!

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Adds two new fish, found very rarely from the stasis. Also planned is a new adventure through exploration drones that will give you a guaranteed random syndifish

The Donkfish is a fish created by Donk Co as a delicious snack. Unfortunately, the final product made testers hurl. So now they're something more of a company mascot! Shame, because it LOOKS like a chicken nugget.

image

The Emulsijack is a fish generally created by the syndicate. It's vile, disgusting, and releases a toxin into the waters it can live in (hint: BOTH fresh and saltwater) that will kill other fish.

image

## Why It's Good For The Game

A little bit of fish variety and I wanted a new special outlet to get them. Also, special fish that have special powers that must be paid attention to!

## Changelog

:cl:
add: Added two new fish to the aquarium
/:cl:

### CODE REVIEW

File: code/modules/explorer_drone/loot.dm [hunk 1]
```diff
@@ -86,6 +86,11 @@ GLOBAL_LIST_INIT(adventure_loot_generator_index,generate_generator_index())
 	id = "weapons"
 	loot_list = list(/obj/item/gun/energy/laser,/obj/item/melee/baton/security/loaded)
 
+/// Rare fish! Of the syndicate variety
+/datum/adventure_loot_generator/simple/syndicate_fish
```
Comment (tralezab) [inline code/modules/explorer_drone/loot.dm:90]: this isn't used in my current pr but i want to make an adventure with this

**Comment by Ghilker** [comment]
This should be expansion and not mechanic

**Comment by tralezab** [comment]
so be it

**Comment by optimumtact** [comment]
Finally

**Pull request merged**